src/style.css

```diff
@@ -1,3 +1,72 @@
+/* stylelint-disable rule-empty-line-before */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
 body {
   margin: 0;
 }
+
+.link {
+  text-decoration: none;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)